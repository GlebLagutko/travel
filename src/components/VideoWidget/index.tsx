import React from 'react';
import {createStyles, makeStyles, Theme} from '@material-ui/core/styles';

const useStyles = makeStyles((theme: Theme) =>
    createStyles({
        root: {
            minWidth: "300px",
            height: "330px",
            width: '70%',
            maxWidth: "1000px"
        }


    }),
);

export default function VideoWidget({video}) {

    const classes = useStyles();

    return (
        <iframe className={classes.root} src={video} frameBorder="0"
                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
                allowFullScreen/>
    );
}
