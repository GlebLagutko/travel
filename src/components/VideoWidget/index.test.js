import {create} from 'react-test-renderer';

import React from "react";
import VideoWidget from ".";

describe('VideoWidget', () => {
  it('renders correctly', async () => {


    const testRenderer = create(
      <VideoWidget video="http://test.vodeo"/>
    );

    expect(testRenderer.toJSON()).toMatchSnapshot();

  });

});