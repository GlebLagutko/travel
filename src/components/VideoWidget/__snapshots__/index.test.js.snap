// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoWidget renders correctly 1`] = `
<iframe
  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
  allowFullScreen={true}
  className="makeStyles-root-1"
  frameBorder="0"
  src="http://test.vodeo"
/>
`;
